Containerfile.redis

```diff
@@ -1,39 +1,21 @@
-# Use Red Hat Universal Base Image (UBI) 9 as the base image
-FROM registry.access.redhat.com/ubi9/ubi-minimal
+# Use CentOS Stream 9 as the base image
+FROM quay.io/centos/centos:stream9
 
 # Install necessary dependencies (gcc, make, etc.) for building Redis from source
-RUN microdnf install -y gcc make wget tar gzip vim openssl openssl-devel tcl && \
-    microdnf clean all
-
-# Set the Redis version
-ARG REDIS_VERSION=7.2.4
-
-# Download and extract Redis source code
-RUN mkdir -p /etc/redis && \
-    wget https://download.redis.io/releases/redis-${REDIS_VERSION}.tar.gz && \
-    tar xzf redis-${REDIS_VERSION}.tar.gz -C /etc/redis --strip-components=1 && \
-    rm redis-${REDIS_VERSION}.tar.gz && \
-    chmod 777 /var/run
+RUN yum install -y gcc make wget tar gzip vim openssl openssl-devel tcl && \
+    yum clean all && mkdir -p /etc/redis && \
+    wget https://download.redis.io/releases/redis-7.2.4.tar.gz && \
+    tar xzf redis-7.2.4.tar.gz -C /etc/redis --strip-components=1 && \
+    rm redis-7.2.4.tar.gz && \
+    chmod 770 /var/run
 
 # Change working directory to the extracted Redis source code directory
 WORKDIR /etc/redis
 
 # Build Redis from source
 RUN make BUILD_TLS=yes && \
-    make install
-
-# Grant full permissions
-RUN chown -R 777 /etc/redis
-
-# Set protected-mode to no and bind to all interfaces (0.0.0.0) in redis.conf
-RUN sed -i 's/^protected-mode yes/protected-mode no/' redis.conf
-RUN sed -i 's/^bind 127.0.0.1 -::1/bind 0.0.0.0/' redis.conf
-RUN sed -i 's/^port 6379/port 0/' redis.conf
-RUN sed -i 's/^# tls-port 6379/tls-port 6379/' redis.conf
-RUN sed -i 's/^# tls-cert-file redis.crt/tls-cert-file \/etc\/certs\/tls.crt/' redis.conf
-RUN sed -i 's/^# tls-key-file redis.key/tls-key-file \/etc\/certs\/tls.key/' redis.conf
-RUN sed -i 's/^# tls-ca-cert-file ca.crt/tls-ca-cert-file \/etc\/certs\/cm-olsredisca\/service-ca.crt/' redis.conf
-RUN sed -i 's/^# tls-auth-clients optional/tls-auth-clients optional/' redis.conf
```

Comment:
you don't need any of these config file tweaks any more?

Reply:
Those do not look good here, So I am passing them through cli [command](https://github.com/openshift/lightspeed-operator/pull/25/files#diff-0bba25c929831058ca39ea8a7599b3ee823c1eee0798ae2a1a1228665fd33e7dR53).